package.json

```diff
@@ -28,6 +28,12 @@
       "default": false,
       "type": "boolean",
       "description": "When disabled, generated packages are linked into Atom in both normal mode and dev mode. When enabled, generated packages are linked into Atom only in dev mode."
+    },
+    "packageSyntax": {
+      "default": "coffeescript",
```

Comment:
That's almost always the right choice, but if we can push people away from CS and to JS... :smile: